server/db/dbManager.js

```diff
@@ -59,6 +59,22 @@ class UserManager {
 		return 'success';
 	}
 
+	async addRefreshToken (username, refreshToken) {
+		await this.collection.updateOne({ username }, { $set: { refreshToken } });
+	}
+
+	async deleteRefreshToken (refreshToken) {
+		await this.collection.updateOne({ refreshToken }, { $unset: { refreshToken: '' } });
+	}
+
+	async findRefreshToken (refreshToken) {
+		const user = await this.collection.findOne({ refreshToken });
+		if (user) {
+			return user.username;
+		}
+		return false;
```

Comment:
What is the type of the returned value of this function?

Reply:
JS weirdness–type :)

Reply:
I changed it to either the string if found or null